php/Exchange.php

```diff
@@ -32,7 +32,6 @@
 
 use kornrunner\Eth;
 use kornrunner\Secp256k1;
-use kornrunner\Solidity;
```

Comment:
We were using it here: https://github.com/ccxt/ccxt/pull/5246/files#diff-d087d122b9d18390abb24d8b2cb6160fR2542

Reply:
I've restored it in `Exchange.php` ) Otherwise we will also have to cut it away from `composer.json`, but then we will break the sha3.